DIMS/GenerateExcel.R

```diff
@@ -219,10 +221,10 @@ if (z_score == 1) {
   }
 
   cnames_nooutliers <- gsub("_Zscore", "_OutlierRemovedZscore", colnames_z)
-  outlist_nooutliers %>% apply(outlist_nooutliers, 2, function(col) {
-    (as.numeric(as.vector(unlist(col))) - outlist_nooutliers$avg.ctrls) / outlist_nooutliers$sd.ctrls
-  }) %>%
-    cbind(outlist_nooutliers)
+  outlist_nooutliers_zscores <- apply(outlist_nooutliers[, intensity_col_ids, drop = FALSE], 2, function(col) {
+    (as.numeric(col) - outlist_nooutliers$avg.ctrls) / outlist_nooutliers$sd.ctrls
+  }) 
+  outlist_nooutliers <- cbind(outlist_nooutliers, outlist_nooutliers_zscores)
```

Comment:
In a previous code change in this file (line 128) you included tidy verse functions, and here you remove them. I would suggest to keep them here, for consistency.
```suggestion
  outlist_nooutliers_zscores <- outlist_nooutliers %>%
     select(all_off(intensity_col_ids)) %>% # possibly without all_of()
     apply(MARGIN=2, function(col) {
       (as.numeric(col) - outlist_nooutliers$avg.ctrls) / outlist_nooutliers$sd.ctrls
  }) 
  outlist_nooutliers <- cbind(outlist_nooutliers, outlist_nooutliers_zscores)
```

Or, if you want to make it fancy:
```suggestion
outlist_nooutliers_zscores <- outlist_nooutliers %>%
  mutate(across(all_of(intensity_col_ids),
                ~ (as.numeric(.) - avg.ctrls) / sd.ctrls))
```

Reply:
Yes, I changed back to tidy verse and back because when using tidy verse the code changed to having a different outcome. It did change from a for loop to apply.